src/components/Editable.tsx

```diff
@@ -127,6 +127,7 @@ const Editable = ({
   const nullRef = useRef<HTMLInputElement>(null)
   const contentRef = editableRef || nullRef
   const editingOrOnCursor = useSelector(state => state.editing || equalPath(path, state.cursor))
+  const isPressingRef = useRef(false)
```

Comment:
I'm sure you'll get to this, but let's make sure to add a detailed comment here explaining the purpose of `isPressingRef`.